src/migrations/20240917071436-add-foreign-key-to-role-permissions.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        UPDATE role_permission SET environment = null where environment = '';
```

Comment:
Update all permissions in `role_permissions` where environment is set to empty string